recipe/meta.yaml

```diff
@@ -58,17 +58,17 @@ requirements:
     - azure-identity-cpp
     - azure-storage-blobs-cpp
     - bitmagic
-    - krb5 =1.20.1
+    - krb5
     - xxhash
     - semimap
     - grpcio-tools
     - protobuf
     - spdlog
     - robin_hood
     - recycle
-    - lmdb ==0.9.22
+    - lmdb
     - lmdbxx ==0.9.14
-    - fmt <10
+    - fmt
```

Comment:
I didn't add the fmt 10 migrator here yet, because I don't know why this cap is there. In any case, the pinning is still at 9; but we should eventually be able to follow suit.

Reply:
ArcticDB is currently incompatible with fmt 10, and I agree that we need it to.